tf-dataflow/README.md

```diff
@@ -196,11 +196,11 @@ In case real world data exists and needed to be convert as well:
 
 ### Phase 3: Use Google AI Platform to train an object detection model
 
-#### *Submit a training job*
+In this section we will provide example configuration and commands to train a built-in object detection model on Google AI Platform. To view the official document for Google AI Platform, click [HERE](https://cloud.google.com/ai-platform/training/docs/algorithms/object-detection).
 
-Submit a built-in object deteciton training job by follow instructions [here](https://cloud.google.com/ai-platform/training/docs/algorithms/object-detection).
+#### *Submit a training job*
 
-Training job can be submitted either through GCP Console UI or Google Cloud SDK (`gcloud`).
+Training job can be submitted either through GCP Console UI or Google Cloud SDK (`gcloud`). Using `gcloud` command is recommended because one can easily iterate on it without re-entering all parameters over and over again. To install and initialize `gcloud` on your working environment, follow Google's [official instruction](https://cloud.google.com/sdk/docs/install).
```

Comment:
I can add the gcloud installation instructions at the top? I don't think we need them here. @mkamalza - opinions?

Reply:
I agree we do not need full instructions on installing gcloud here. It should be sufficient to:
- Point users to where instructions are (provided by google)
- Emphasize that users need to be logged in and have the correct project selected